### ISSUE TITLE
Office.context.ui.displayDialogAsync with a SharedRuntime creates an empty dialog / overlay

### ISSUE THREAD

**Issue by SeanSmithApryse** [OfficeDev/office-js#2979]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

# Office.context.ui.displayDialogAsync with a SharedRuntime creates an empty dialog / overlay

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: Office on the web
* Host [Excel, Word, PowerPoint, etc.]: Word
* Office version number: 16.0.15902.41009
* Operating System: Windows
* Browser (if using Office on the web): Chrome + Edge
 
## Expected behavior
An empty blank dialog is not created and the user can continue using the app

## Current behavior
An empty blank dialog is created when displayDialogAsync is called, with an overlay that prevents anymore interaction with the app

## Steps to reproduce

1. yo office --projectType taskpane --name "my office add in" --host word --js true
2. Replace the manifest.xml taskpane.js webpack.config in the created folder with the files in the attached zip.
3. Drop login.html into the folder from the attached zip.
4. Build / run dev-server / add the manifest
5. Click "Toggle Taskpane"
6. Click "Open Dialog"
[Files.zip](https://github.com/OfficeDev/office-js/files/10026050/Files.zip)

# Provide additional details

1. Doesn't happen if I reconfigure the addin to not be in a SharedRuntime

## Context

I am trying to create a SharedRuntime addin that authenticates to Graph in the way demonstrated in this sample (using a dialog outside of the iframe)
https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/auth/Office-Add-in-Microsoft-Graph-React
When the User is prompted for permission to display the dialog, an overlay and dialog takes over the screen preventing input.
![image](https://user-images.githubusercontent.com/99692264/202305564-5bcf6299-7a4c-4c29-928c-84aed26a1c83.png)

## Useful logs

No errors reported in console during issue

**Comment by RuizhiSunMS** [comment]
@madhavagrawal17 for sharedruntime.
And @Rick-Kirkham do you know who own dialog api now?

**Comment by SeanSmithApryse** [comment]
Hello - are there any updates on this issue?

**Comment by IPetersonPDFTron** [comment]
@yihuax-zz Any updates on this issue?

**Comment by mbinsfeld-pdftron** [comment]
Hi,
Are there any updates on this issue? My team has been blocked on this since November.

**Comment by mbinsfeld-pdftron** [comment]
Any explanation for why this was closed? Is there some other way that what we're trying to do is possible or is it just not supported?

**Comment by gergzk** [comment]
We've reopened this one and the dev team is taking a look.

**Comment by dariomariani** [comment]
Hello, my team is encountering the same issue and is completely blocked. This issue has a serious business impact because the dialog trying to be opened is a login dialog against our backend, and we are forced to unsupport Web addins.

The behaviour, the steps to reproduce and the platform is exactly the same as the original post.
